js/apps/admin-ui/src/dashboard/Dashboard.tsx

```diff
@@ -67,48 +69,46 @@ const EmptyDashboard = () => {
   );
 };
 
+const FeatureItem = ({ feature }: { feature: FeatureRepresentation }) => {
+  const { t } = useTranslation("dashboard");
+  return (
+    <ListItem key={feature.name} className="pf-u-mb-sm">
+      {feature.name}{" "}
+      {feature.type === FeatureType.EXPERIMENTAL ? (
+        <Label color="orange">{t("experimental")}</Label>
+      ) : null}
+      {feature.type === FeatureType.PREVIEW ? (
+        <Label color="blue">{t("preview")}</Label>
+      ) : null}
+      {feature.type === FeatureType.DEFAULT ? (
+        <Label color="green">{t("supported")}</Label>
+      ) : null}
```

Comment:
There should be no need to use a ternary expression here:
```suggestion
      {feature.type === FeatureType.EXPERIMENTAL && (
        <Label color="orange">{t("experimental")}</Label>
      )}
      {feature.type === FeatureType.PREVIEW && (
        <Label color="blue">{t("preview")}</Label>
      )}
      {feature.type === FeatureType.DEFAULT && (
        <Label color="green">{t("supported")}</Label>
      )}
```